src/sentry/notifications/manager.py

```diff
@@ -147,29 +129,40 @@ def update_settings(
             if team is not None:
                 actor = RpcActor.from_object(team)
         assert actor
+        assert team or user, "Both user and team cannot be none"
```

Comment:
Should there be an assertion for `not (team and user)` to prevent the scenario where we make a mistake and set both?